Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -528,20 +528,20 @@ ErrorOr<i32> HuffmanTable::read_symbol_non_oob(BigEndianInputBitStream& stream)
 
 class Symbol : public RefCounted<Symbol> {
 public:
-    static NonnullRefPtr<Symbol> create(NonnullOwnPtr<BilevelImage> bitmap)
+    static NonnullRefPtr<Symbol> create(NonnullRefPtr<BilevelImage> bitmap)
```

Comment:
I'm very supportive of this commit; I almost made the change myself once.

Having said that, the point of `Symbol` is to be a RefCounted wrapper around BilevelImage, so after this commit this class is iirc pretty pointless.

But I can clean it up in a separate commit later on.

(No action needed.)

Reply:
-> #26253